src/components/MainSidebar/ProgramDimensionsPanel/useProgramDataDimensions.js

```diff
@@ -79,12 +79,12 @@ const createDimensionsQuery = ({
     dimensionType,
     nameProp,
 }) => {
-    const resource =
-        inputType === OUTPUT_TYPE_TRACKED_ENTITY
-            ? 'analytics/trackedEntities/query/dimensions'
-            : inputType === OUTPUT_TYPE_ENROLLMENT
-            ? 'analytics/enrollments/query/dimensions'
-            : 'analytics/events/query/dimensions'
+    const resourceMap = {
```

Comment:
This object could be outside any function right? Then it is only created once.

Reply:
moved 👍